src/App.jsx

```diff
@@ -2,17 +2,16 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
 
   const addOne = () => {
-    // write code here
+    setCount(getCount => getCount + 1);
```

Comment:
Please don't use verb in name for this arg, because this is the previous state
```suggestion
    setCount(prevCount => prevCount + 1);
```